assets/src/main/java/bisq/asset/MileAddressValidator.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset;
+
+import org.bitcoinj.core.AddressFormatException;
+import org.bitcoinj.core.Base58;
+
+import java.util.Arrays;
+import java.util.zip.CRC32;
+import java.util.zip.Checksum;
+
+/**
+ * Mile address - base58(32 bytes of public key + 4 bytes of crc32)
+ */
+public class MileAddressValidator implements AddressValidator {
```

Comment:
Make this inner class of Mile as it will not be used anywhere else.

Reply:
Fixed. I had made like I've seen in Grin pull request